packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -186,7 +189,8 @@
       } else {
         approved = true;
       }
-      isPaid = requestData?.balance?.balance! >= requestData?.expectedAmount;
+
+      status = checkStatus(requestData || request);
```

Comment:
@sstefdev why checkStatus is called above and here with same params? It seems redundant. Could explain please? Cheers

Reply:
We need an initial value, and then we check again when we call the function.

Reply:
@rodrigopavezi Basically we want the value to be svelte reactive